cmd/sealer/cmd/cluster/run.go

```diff
@@ -110,6 +111,16 @@ func NewRunCmd() *cobra.Command {
 }
 
 func runWithClusterfile(clusterFile string, runFlags *types.RunFlags) error {
+	var p2p bool
+	switch runFlags.Distributor {
+	case "sftp":
+		p2p = false
+	case "p2p":
+		p2p = true
+	default:
```

Comment:
default use sftp

Reply:
The default is `sftp`, when nothing is given. However if the user enters something other than "sftp" or "p2p", there should be an error message rather than continuing anyway.

Reply:
that's OK